cmake/FindOpenMP_Offloading.cmake

```diff
@@ -0,0 +1,75 @@
+#  OpenMP_Offloading - finds flags for OpenMP Offloading
+#
+#  requires 1. find_package(OpenMP REQUIRED) call before
+#           2. DEVICE_BACKEND must be set
+#           3. DEVICE_ARCH must be set
+#
+#  OpenMP_Offloading_FOUND               - compiler is able to perform OpenMP offloading
+#  OpenMP_Offloading_FLAGS               - compiler flags for OpenMP Offloading
+#  OpenMP::OpenMP_CXX_with_offloading    - imported target
+#
+#  NOTE: it is not an official cmake search file
+#  authors: Ravil Dorozhinskii
+#  email: ravil.dorozhinskii@tum.de
+#
+
+include(FindPackageHandleStandardArgs)
+include(CheckCXXSourceRuns)
+include(CMakePushCheckState)
+
+cmake_push_check_state()
+
+set(_PARENT_CMAKE_CXX_STANDARD ${CMAKE_CXX_STANDARD})
```

Comment:
Why is this needed? We are using CMAKE_CXX_STANDARD 17 anyway?

Reply:
I guess I can remove `CMAKE_CXX_STANDARD` from this file. I started with a copy of `FindFILESYSTEM.cmake` file and forgot to remove it. 

In case of `FindFILESYSTEM.cmake`, _in general_, one doesn't know whether  `CMAKE_CXX_STANDARD 17` has already been set. It was just a generalization in case of somebody decides to re-use the file for other sub-projects.